templates/plugin/gutterball/foreman_gutterball.yaml.erb

```diff
@@ -0,0 +1,2 @@
+:foreman_gutterball:
+  :url: <%= @protocol %>://<%= @fqdn %>:<%= @port %><%= @path %>
```

Comment:
one last thing...

let's stick with the keys being strings instead of symbols, so we can stay uniform with other configs.

so
```erb
foreman_gutterball:
  url: <%= @protocol %>://<%= @fqdn %>:<%= @port %><%= @path %>
```